src/scenes/Esittelijat/components/PaatetytAsiat.js

```diff
@@ -33,11 +33,11 @@ const PaatetytAsiat = () => {
     return !!muutospyynnot.paatetyt && muutospyynnot.paatetyt.fetchedAt
       ? generatePaatetytAsiatTableStructure(
           muutospyynnot.paatetyt.data,
-          intl.formatMessage,
+          intl,
           history
         )
       : [];
-  }, [history, intl.formatMessage, muutospyynnot.paatetyt]);
+  }, [history, intl, muutospyynnot.paatetyt]);
```

Comment:
formatMessage is the only method / property that generatePaatetytAsiatTableStructure needs. Why should it take the whole intl object as a parameter?

Reply:
locale is needed when showing organization name and region